flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/job/AsynchronousJobOperationKey.java

```diff
@@ -33,7 +33,7 @@
  * A pair of {@link JobID} and {@link TriggerId} used as a key to a hash based
  * collection.
  *
- * @see AbstractAsynchronousOperationHandlers.CompletedOperationCache
+ * @see AbstractAsynchronousOperationHandlers
```

Comment:
This reference wasn't updated properly as it no longer points to the cache.

Reply:
It's not possible to reference the cache anymore because it has default visibility now. I also don't think it is necessary to reference it here to understand how `AsynchronousJobOperationKey` is used.